src/Picker/ArticlePickerProvider.php

```diff
@@ -82,7 +82,7 @@ protected function getLinkClass()
     /**
      * {@inheritdoc}
      */
-    protected function getRouteParameters(PickerConfig $config)
+    protected function getRouteParameters(PickerConfig $config = null)
```

Comment:
Because the core pickers are not picking from subtables. News, Event and FAQ pickers do.